src/cook-web/src/components/cm-editor/util.ts

```
@@ -22,6 +22,14 @@ const biliVideoContextRegexp =
 const agiImgContextRegexp =
   /<span\s+data-tag="image"[^>]*data-url="([^"]*)"[^>]*data-title="([^"]*)"[^>]*data-scale="([^"]*)"[^>]*>([^<]+)<\/span>/gi
 
+const getProfileKeyListFromContent = (content: string): string[] => {
+  const keys = new Set<string>()
+  for (const match of content?.matchAll(profileRegexp)) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)